Examples/Modules/RigidInjection/analysis_rigid_injection_LabFrame.py

```diff
@@ -23,6 +23,7 @@
 import sys
 import yt
 import numpy as np
+import os
```

Comment:
To be PEP8 compliant, the `import os` should be before the imports of numpy and yt.

Reply:
Interesting style choice, because contrary to C++ this has no semantic difference:
https://www.python.org/dev/peps/pep-0008/#imports

I think we need a bot for this, maybe we can add this to `.pre-commit-config.yaml`

Reply:
Clarified on Slack: separate, we are  looking for tools that can do it :)

Maybe `autopep8` or `flake8`.

Reply:
I tried `autopep8 --select E401,E402 --in-place --aggressive --aggressive <filename>` and all it does is sort alphabetically.

Reply:
The tool `isort` does the trick: https://github.com/ECP-WarpX/WarpX/pull/2686
https://github.com/PyCQA/isort